kubernetes/mlflow-tracking-operator-master/examples/kubernetes-manifests/example-app.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mlflow-example-deployment
+  labels:
+    app: mlflow-example
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: mlflow-example
+  template:
+    metadata:
+      labels:
+        app: mlflow-example
+    spec:
+      containers:
+      - name: mlflow-example
+        image: quay.io/zmhassan/mlflow-example
+        env:
+        - name: MLFLOW_S3_ENDPOINT_URL
+          value: "http://10.110.217.77:5000"
```

Comment:
In kubernetes we can either use configmaps which is like key/value pair file. Or secret.

Here is an example:

```yaml 
apiVersion: v1
kind: Pod
metadata:
  name: secret-env-pod
spec:
  containers:
  - name: mycontainer
    image: redis
    env:
      - name: SECRET_USERNAME
        valueFrom:
          secretKeyRef:
            name: mysecret
            key: username
      - name: SECRET_PASSWORD
        valueFrom:
          secretKeyRef:
            name: mysecret
            key: password
  restartPolicy: Never
```

Reply:
So that part to keep an eye on is that the secret has a key/value. So when you first create your secret make sure you specify it. You can inject all values from a secret if you want all keys injected. We do that currently.